doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -0,0 +1,106 @@
+
+image:ai/riskrating.png[]
+
+* *Cardholder Risk Rating*
++
+Joe is a Silver cardholder over the age of 25. He has had more than two previous disputes, so he has a risk rating of 2. If Joe had no previous disputes, he would have a risk rating of 0.
+
+* *Dispute Risk Rating*
++
+Because Joe is a Silver cardholder and the disputed amount is 40 dollars, Joe has a rating of 1 on the *Dispute Risk Rating* table. If the disputed amount was 140 dollars, Joe would have a risk rating of 2.
+
+The following formula, which is implemented as part of the *Process Automatically* final decision in the DMN model, uses the scores from the two decision tables to determine whether to automatically charge back the disputed amount (*Milestone 1*) or whether more investigation is needed (*Milestone 2*).
+[source]
+----
+(Cardholder Risk Rating + Dispute Risk Rating) < 5
+----
+If Joe's overall risk score is less than five, his disputed amount is charged back automatically (*Milestone 1*). If the overall score is five or greater, then his dispute is processed manually (*Milestone 2*).
+
+== Adding machine learning to {PRODUCT} digital decisioning
+Because Fortress Bank has historical data about its customers, including previous transaction and dispute history, the bank can use that data with machine learning to create predictive models that can be used in the DMN model decision tasks. This results in a more accurate assessment of the risk compared to the decision tables created by business analysts.
+
+Fortress Bank has two sets of PMML files that contain models which more accurately assess risk predictions. One set is based on the linear regression algorithm and the other is based on the random forests algorithm.
+
+image:ai/pmml-models.png[]
+
+Linear regression is one of the most widely used algorithms in both statistics and machine learning. It uses a linear equation that combines a set of numeric input and output values. Random forests use many decision trees as inputs to create prediction models.
+
+.Adding PMML files
+Michelle imports the *dispute_risk_linear_regression* PMML file into her project. She adds the *Cardholder Risk Model* business model knowledge node to the DMN model and associates the *dispute_risk_linear_regression* PMML file with the node. {PRODUCT} analyzes the PMML file and add input parameters to the node. Michelle associates the *Cardholder Risk Model* node with the *Dispute Risk Rating*.
+
+Michelle then adds the *credit_card_holder_risk_linear_regression* PMML model to the projects, creates the *Dispute Risk Model* mode DMN file, creates, and associates the *credit_card_holder_risk_linear_regression* PMML file with the node. {PRODUCT} analyzes the PMML file and add input parameters to the node.
```

Comment:
Please remove the comma after "creates" in the following sentence: 

"the *Dispute Risk Model* mode DMN file, creates, and associates the *credit_card_holder_risk_linear_regression* PMML file with the node."

Reply:
Done